packages/core/src/components/ModalNew/Modal/__tests__/Modal.test.tsx

```diff
@@ -27,6 +28,16 @@ describe("Modal", () => {
     expect(getByTestId("modal")).toHaveAttribute("aria-modal", "true");
   });
 
+  it("does not render when 'show' is false", () => {
```

Comment:
```suggestion
  it("should not render when 'show' is false", () => {
```

Reply:
I've wrote all the tests not in a `should` style (by mistake, accidentally ignored Gebeta's chosen style)
planning on changing all the tests names later 😓